libs/SalesforceSDKCore/Configuration/SalesforceSDKCore-Common.xcconfig

```diff
@@ -13,5 +13,5 @@ PRODUCT_NAME = SalesforceSDKCore
 
 OTHER_CFLAGS = -DDDAbstractDatabaseLogger=SF_DDAbstractDatabaseLogger -DDDLog=SF_DDLog -DDDAbstractLogger=SF_DDAbstractLogger -DDDASLLogger=SF_DDASLLogger -DDDAbstractLogger=SF_DDAbstractLogger -DDDLogger=SF_DDLogger -DDDLogFileManager=SF_DDLogFileManager -DDDLogFileManagerDefault=SF_DDLogFileManagerDefault -DDDLogFileFormatterDefault=SF_DDLogFileFormatterDefault -DDDLogFormatter=SF_DDLogFormatter -DDDFileLogger=SF_DDFileLogger -DDDLogFileInfo=SF_DDLogFileInfo -DDDRegisteredDynamicLogging=SF_DDRegisteredDynamicLogging -DDDLogMessage=SF_DDLogMessage -DDDLoggerNode=SF_DDLoggerNode -DDDTTYLogger=SF_DDTTYLogger -DDDTTYLoggerColorProfile=SF_DDTTYLoggerColorProfile -DDDMultiFormatter=SF_DDMultiFormatter -DDDContextWhitelistFilterLogFormatter=SF_DDContextWhitelistFilterLogFormatter -DDDContextBlacklistFilterLogFormatter=SF_DDContextBlacklistFilterLogFormatter -DDDLoggingContextSet=SF_DDLoggingContextSet -DDDDispatchQueueLogFormatter=SF_DDDispatchQueueLogFormatter -DDDExtractFileNameWithoutExtension=SF_DDExtractFileNameWithoutExtension -DDDLogMessageCopyFile=SF_DDLogMessageCopyFile -DDDLogMessageCopyFunction=SF_DDLogMessageCopyFunction -DDDLogMessageOptions=SF_DDLogMessageOptions -DdoesAppRunInBackground=SF_doesAppRunInBackground
 
-HEADER_SEARCH_PATHS = $(inherited)
+HEADER_SEARCH_PATHS = "$(BUILT_PRODUCTS_DIR)/Headers" "$(TARGET_BUILD_DIR)/Headers" "$(PROJECT_DIR)/../../external/CocoaLumberjack/Classes/**" $(inherited)
```

Comment:
That seems to be true: I made the change here: https://github.com/wmathurin/SalesforceMobileSDK-iOS/commit/7b87b9cd5ff2eec1d9e9b4b30c694e4bd08a07ed